src/aces/aggregate.py

```diff
@@ -305,71 +315,74 @@ def aggregate_event_bound_window(
         ...     "is_A": [1, 0, 1, 1, 1, 1, 0, 0],
         ...     "is_B": [0, 1, 0, 1, 0, 1, 1, 1],
         ...     "is_C": [0, 1, 0, 0, 0, 1, 0, 1],
+        ...     "_EVENT_INDEX": [0, 1, 2, 0, 1, 2, 3, 4],
         ... })
-        >>> aggregate_event_bound_window(df, ToEventWindowBounds(True, "is_C", True, None))
+        >>> aggregate_event_bound_window(df, ToEventWindowBounds(True, "is_C", True, None)).drop("timestamp")
```

Comment:
I would keep the "timestamp" column in these tests, and just set the `pl.Config` params to print wider dataframes.

Reply:
Actually I did this more to satistfy the pre-commit rules of lines not extending past 120 characters - should doctests still follow these rules? What's the best way to go around this?